_stbt/core.py

```diff
@@ -1241,11 +1244,8 @@ def wait_until(callable_, timeout_secs=10, interval_secs=0, stable_secs=0):
     """
     import time
 
-    class NoValue(object):
-        pass
-
-    stable_value = NoValue
-    stable_since = None
+    stable_value = None
+    stable_since = time.time()
```

Comment:
I think this `time.time()` is misleading.  According to my reading of the function the value will never be used.  You could leave it as `None` and change the code below (see comment below)